### PR TITLE
[WIP] Access your data for all websites

### PR DESCRIPTION
**Pull request by championshuttler**

### CODE REVIEW

**Comment by tsl143** [comment]
Please resolve the conflicts.
The url param has been updated with generic URL, please check.

File: permissions.js [hunk 1]
```diff
@@ -35,22 +35,22 @@ const permissionsGlossary = {
     },
     "telemetry": {
         text: browser.i18n.getMessage("telemetryText"),
-        url: browser.i18n.getMessage("noURL")
+        url: "No"
```
Comment (tsl143) [inline permissions.js]: Needs to be `no` smallcase as its used in `if` condition .

File: permissions.js [hunk 2]
```diff
@@ -35,22 +35,22 @@ const permissionsGlossary = {
     },
     "telemetry": {
         text: browser.i18n.getMessage("telemetryText"),
```
Comment (tsl143) [inline permissions.js]: rebase against master, there are changes

**Comment by championshuttler** [comment]
image
 

Can not access this PR because i unforked the repo and forked again :stuck_out_tongue: So closing it now.

**Comment by championshuttler** [comment]
image
 

Can not access this PR because i unforked the repo and forked again :stuck_out_tongue: So closing it now.

**Pull request closed without merging**